drivers/compaq-mib.c

```diff
@@ -84,87 +84,87 @@
 
 /* Not used, as no longer supported by MIB ver. 1.76 (Github issue 118)
 static info_lkp_t cpqpower_alarm_ob[] = {
-	{ 1, "OB" },
-	{ 0, NULL }
+	{ 1, "OB", NULL, NULL },
+	{ 0, NULL, NULL, NULL }
 };
 */
 
 /* Not used, as no longer supported by MIB ver. 1.76 (Github issue 118)
 static info_lkp_t cpqpower_alarm_lb[] = {
-	{ 1, "LB" },
-	{ 0, NULL }
+	{ 1, "LB", NULL, NULL },
+	{ 0, NULL, NULL, NULL }
 };
 */
 
 /* Defines for CPQPOWER_OID_POWER_STATUS (1) */
 static info_lkp_t cpqpower_pwr_info[] = {
-	{ 1, ""       /* other */ },
-	{ 2, "OFF"    /* none */ },
-	{ 3, "OL"     /* normal */ },
-	{ 4, "OL BYPASS" /* bypass */ },
-	{ 5, "OB"     /* battery */ },
-	{ 6, "OL BOOST"  /* booster */ },
-	{ 7, "OL TRIM"   /* reducer */ },
-	{ 8, "OL"   /* parallelCapacity */ },
-	{ 9, "OL"   /* parallelRedundant */ },
-	{ 10, "OL" /* HighEfficiencyMode */ },
-	{ 0, NULL }
+	{ 1, ""       /* other */, NULL, NULL },
+	{ 2, "OFF"    /* none */, NULL, NULL },
+	{ 3, "OL"     /* normal */, NULL, NULL },
+	{ 4, "OL BYPASS" /* bypass */, NULL, NULL },
+	{ 5, "OB"     /* battery */, NULL, NULL },
+	{ 6, "OL BOOST"  /* booster */, NULL, NULL },
+	{ 7, "OL TRIM"   /* reducer */, NULL, NULL },
+	{ 8, "OL"   /* parallelCapacity */, NULL, NULL },
+	{ 9, "OL"   /* parallelRedundant */, NULL, NULL },
+	{ 10, "OL" /* HighEfficiencyMode */, NULL, NULL },
```

Comment:
Note: here and in a couple other drivers, we had comments about the field inside its table entry (not following it like elsewhere); at the moment I did not bother to reformat these cosmetics amid the large codebase cleanup, but eventually someone might be welcome to do so.